split.go

```diff
@@ -0,0 +1,31 @@
+package goyek
+
+// SplitTasks splits command line arguments into tasks and the rest.
+// Tasks are identified as non-flag arguments at the beginning.
+// The rest includes flags and any arguments after "--".
+//
+// This function does not parse flags - it only separates tasks from flags/args.
+// To parse flags, use flag.Parse() or similar with the returned rest slice.
```

Comment:
```suggestion
// To parse flags, use flag.CommandLine.Parse() or similar with the returned rest slice.
```

Or something like `FlagSet.Parse`. `flag.Parse()` itself doesn't allow providing a slice.

Also either here, the example, or both, it would be good to mention something like `flag.CommandLine.Args` will return any positional arguments after a `--` as it wasn't obvious to me until trying it out.

Reply:
Addressed b5cf398